ManageWikiSettings.php

```diff
@@ -1182,6 +1182,15 @@
 		'help' => 'Open collapsed groups when the screen is narrow.',
 		'requires' => [],
 	],
+	'wgShortDescriptionEnableTagline' => [
+		'name' => 'ShortDescription Enable Tagline',
+		'from' => 'shortdescription',
+		'type' => 'check',
+		'overridedefault' => true,
+		'section' => 'parserfunctions',
+		'help' => 'Enables short descritption in site tagline',
+		'requires' => [],
```

Comment:
This would need 'requires' => [
			'extensions' => [ as it wouldn't make sense to use this setting without the extension being enabled.